packaging/document_type/postinst

```diff
@@ -0,0 +1,9 @@
+if [ -f /etc/irods/service_account.config ] ; then
+  . /etc/irods/service_account.config
+else
+  IRODS_SERVICE_ACCOUNT_NAME=`stat --format "%U" /var/lib/irods`
+  IRODS_SERVICE_GROUP_NAME=`stat --format "%G" /var/lib/irods`
+fi
+
+
+#chown $IRODS_SERVICE_ACCOUNT_NAME:$IRODS_SERVICE_GROUP_NAME /var/lib/irods
```

Comment:
so we don't want this to do any healing on CentOS?

Reply:
this package consists only of root-owned files/directories under /usr/lib/irods, so there's no need at present. As to whether this postinst (which was created in the process of appropriating the similar [mechanism](https://github.com/irods/irods_capability_storage_tiering/blob/49011e7c70c03dfdf5bf3f303882b87d349cff3a/unified_storage_tiering.cmake#L94) within the irods storage tiering repo) technically needs to exist at all, I need to ask someone more expert in things CMake.